packages/apidom-ls/src/config/openapi/header/lint/schema--mutually-exclusive.ts

```diff
@@ -2,15 +2,15 @@ import { DiagnosticSeverity } from 'vscode-languageserver-types';
 
 import ApilintCodes from '../../../codes';
 import { LinterMeta } from '../../../../apidom-language-types';
-import { OpenAPI3 } from '../../target-specs';
+import { OpenAPI30 } from '../../target-specs';
 
 const schemaMutuallyExclusiveLint: LinterMeta = {
   code: ApilintCodes.OPENAPI3_0_HEADER_FIELD_SCHEMA_MUTUALLY_EXCLUSIVE,
   source: 'apilint',
   message: 'The `schema` field and `content` field are mutually exclusive.',
   severity: DiagnosticSeverity.Error,
   linterFunction: 'missingFields',
-  linterParams: [['schema']],
+  linterParams: [['schema', 'boolean']],
```

Comment:
IMHO all changes in this file should be reverted. There is somenting fishy going on in this file vs [schema--mutually-exclusive-openapi-3-1-asyncapi-2.ts](https://github.com/swagger-api/apidom/pull/3640/files#diff-48cac0ce9d6f66df41c2a03314d9338623db4106a0d57e242e7191b2785c2855)

```suggestion
  linterParams: [['schema']],
```